Dockerfile

```diff
@@ -59,24 +36,24 @@ RUN go install github.com/gogo/protobuf/protoc-gen-gogofaster
 RUN go install github.com/gogo/protobuf/protoc-gen-gogoslick
 RUN go install github.com/gogo/protobuf/protoc-gen-gogotypes
 RUN go install github.com/gogo/protobuf/protoc-gen-gostring
-RUN go get github.com/chrusty/protoc-gen-jsonschema/cmd/protoc-gen-jsonschema
 RUN go install github.com/chrusty/protoc-gen-jsonschema/cmd/protoc-gen-jsonschema
-RUN go install github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway
+RUN go install github.com/grpc-ecosystem/grpc-gateway/v2/protoc-gen-grpc-gateway
 RUN go install github.com/envoyproxy/protoc-gen-validate
 RUN go install github.com/mwitkow/go-proto-validators/protoc-gen-govalidators
 RUN go install github.com/pseudomuto/protoc-gen-doc/cmd/...
 RUN go install github.com/infobloxopen/protoc-gen-preprocess
-RUN go install  \
-      -ldflags "-X github.com/infobloxopen/protoc-gen-gorm/plugin.ProtocGenGormVersion=$PGG_VERSION -X github.com/infobloxopen/protoc-gen-gorm/plugin.AtlasAppToolkitVersion=$AAT_VERSION" \
-      github.com/infobloxopen/protoc-gen-gorm
-# Download all dependencies of protoc-gen-atlas-query-validate
-RUN cd ${GOPATH}/src/github.com/infobloxopen/protoc-gen-atlas-query-validate && dep ensure -vendor-only
 RUN go install github.com/infobloxopen/protoc-gen-atlas-query-validate
-
-# Download all dependencies of protoc-gen-atlas-validate
-RUN cd ${GOPATH}/src/github.com/infobloxopen/protoc-gen-atlas-validate && dep ensure -vendor-only
 RUN go install github.com/infobloxopen/protoc-gen-atlas-validate
 
+# TODO: this should be installed the same way once it is compatible with updated protobuf
+RUN GO111MODULE=on go install  \
```

Comment:
This is using go1.16. Default GO111MODULE setting should do the expected behavior https://blog.golang.org/go116-module-changes

Reply:
It was disabled by default on L14 so that it installs from what's already downloaded (the versions specified in go.mod). Otherwise I was seeing some peculiar package related errors, including every install command requiring the version being specified again.
Instead of organizing the package versions in go.mod, I could explicitly include them in the Dockerfile if that sounds more idiomatic to you (it seemed less readable to me).